Source/WTF/wtf/DateMath.cpp

```diff
@@ -479,6 +492,17 @@ static bool parseLong(const char* string, char** stopPosition, int base, long* r
     return true;
 }
 
+static bool parseLong(std::span<const LChar>& string, int base, long* result)
```

Comment:
So sad since `long` and `int` are the same width on virtually every platform. Maybe this should be a template since the functions are identical except for the range check.

Reply:
Fixed.